BE/app/api/routes/script_gen.py

```
@@ -341,6 +348,116 @@ async def run_planner(
+    except Exception as e:
+        logger.error(f"History fetch failed: {e}", exc_info=True)
+        raise HTTPException(
+            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            detail=f"이력 조회 실패: {str(e)}",
+        )
+
+
+@router.get("/scripts/{topic_request_id}")
+async def get_script_by_id(
+    topic_request_id: str,
+    db: AsyncSession = Depends(get_db),
```

Ruff (0.14.14)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)